packages/database/e2e/query/once.e2e.js

```diff
@@ -181,4 +181,26 @@ describe('database().ref().once()', () => {
     callback.should.be.calledOnce();
     callback.should.be.calledWith({ nuggets: 57 });
   });
+
+  it.only('guarantee to resolve after child_added', async () => {
```

Comment:
```suggestion
  it('guarantee to resolve after child_added', async () => {
```

Reply:
I've left this in for now. As this is the one I need to discuss. The implementation is looking pretty tricky!